bootstrap-gruntwork-installer.sh

```diff
@@ -179,11 +193,10 @@ function create_user_data_folder {
   local -r user_data_folder="$1"
   local -r user_data_folder_owner="$2"
   local -r user_data_folder_readme="$user_data_folder/README.txt"
+  local -r no_sudo="$3"
 
   echo "Creating $user_data_folder as a place to store scripts intended to be run in the User Data of an EC2 instance during boot"
-  sudo mkdir -p "$user_data_folder"
-
-sudo tee "$user_data_folder_readme" > /dev/null <<EOF
+tee /tmp/gruntwork_installer_user_data_folder_readme.txt > /dev/null <<EOF
```

Comment:
Does this not require `sudo` now?

Reply:
I wanted to avoid copy pasting this call, so I did that by using a temporary path to generate the file, and then moving that file to the final destination in the if switch down below.